megamek/src/megamek/common/PlanetaryConditions.java

```diff
@@ -683,14 +691,16 @@ public int getVisualRange(Entity en, boolean targetIlluminated) {
                 return 8;
             }
             return 2;
-        } else if ((weatherConditions == WE_HEAVY_SNOW)
-                || (weatherConditions == WE_HEAVY_RAIN)
-                || (weatherConditions == WE_DOWNPOUR)
+        } else if ((weatherConditions == WE_HEAVY_HAIL)
                 || (weatherConditions == WE_SLEET)
+                || (weatherConditions == WE_HEAVY_SNOW)
+                || (blowingSand && (windStrength >= WI_MOD_GALE))
```

Comment:
Why the wind strength check?

Reply:
You have to have both to create the blowing sand condition.